doc/source/modules/ui.inc

```diff
@@ -28,7 +26,7 @@ Read and Write Data Samples
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 For Python users, it is recommended to first load the data into a common format
-via numpy or pandas and then construct :class:`Event` classes.
+via numpy or pandas and then construct :class:`~.Event` classes.
```

Comment:
Just so I understand, what is this `~.` or `.` for?

Reply:
See [here](http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax)

Reply:
Ok. I only could quickly find the ~ functionality. I still dont get `.`. Is that like the default namespace then?

Reply:
From the same page:

> Also, if the name is prefixed with a dot, and no exact match is found, the target is taken as a suffix and all object names with that suffix are searched. For example, :py:meth:`.TarFile.close` references the tarfile.TarFile.close() function, even if the current module is not tarfile. Since this can get ambiguous, if there is more than one possible match, you will get a warning from Sphinx.

Reply:
Ok thanks!